### PR TITLE
Modification in setupClass to skip testcases rather than throwing exception 

### PR DESCRIPTION
**Pull request by pritisarap12**
Skip the testcase which is throwing an exception on "Exceeding connection limit to NetScaler device"

### CODE REVIEW

File: test/integration/component/test_netscaler_lb_algo.py [hunk 1]
```diff
@@ -76,8 +78,12 @@ def setUpClass(cls):
                 cls.testdata["service_offering"]
             )
         except Exception as e:
-            cls.tearDownClass()
-            raise Exception("Warning: Exception in setUpClass: %s" % e)
+            if cls.exception_string in e:
```
Comment (gauravaradhye) [inline test/integration/component/test_netscaler_lb_algo.py]: Do case insensitive string comparisons to be safe. Change wherever applicable.

**Comment by pritisarap12** [comment]
Done with the review changes.

File: test/integration/component/test_netscaler_lb_algo.py [hunk 2]
```diff
@@ -76,8 +78,12 @@ def setUpClass(cls):
                 cls.testdata["service_offering"]
             )
         except Exception as e:
-            cls.tearDownClass()
-            raise Exception("Warning: Exception in setUpClass: %s" % e)
+            if cls.exception_string.lower() in e.lower():
```
Comment (gauravaradhye) [inline test/integration/component/test_netscaler_lb_algo.py]: e.lower() won't work as e is an Exception object. Convert it to string and then use it.

**Comment by gauravaradhye** [comment]
LGTM, merging.

**Pull request closed without merging**